### DISCUSSION TITLE
The new UI changes are TERRIBLE!

### DISCUSSION THREAD

**Discussion by aradalvand** [community/community#58295] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why do you keep making the goddamn UI worse every few months?! Who's getting paid for this?! What was wrong with the previous interface?! Why did you have to ruin things so spectacularly?! What's wrong with you?????????????????????

**Comment by the-officialjosh** [comment]
The need for periodic UI changes is understandable, but considering the community's negative reception of this specific change, it is possible that the decision might be reconsidered and reverted.

**Reply by aradalvand** [reply]
@the-officialjosh Hopefully. Is there a particular place people are providing feedback about this change that I can check out? I couldn't find anything.

**Reply by the-officialjosh** [reply]
More people are complaining about it on various issues like this one - You could search New UI on the discussion search bar and you will see more. However please mark the earlier reply as the answer for others to be guided here when they have similar issues

**Comment by florton** [comment]
The new UI is super tedious and un-intuitive 😢

**Reply by ryukaizen** [reply]
Accessed GitHub after a gap of a month.
Took me hours to brush up the updated button logos and their positions again.

Just why? Microsoft?

![why_github](https://github.com/community/community/assets/55140313/978c33fc-390e-4308-87ab-1cd98550b541)

**Comment by sina-sparkn** [comment]
Actually I kinda like the new UI design  the only thing that I can't understand is the menu on the left that's useless in my opinion.

**Reply by sina-sparkn** [reply]
I changed my mind the whole left section of Navbar is useless.

**Comment by SebixDrela2** [comment]
You are able to disable this feature in feature preview section.

![obraz](https://github.com/community/community/assets/107455395/02f6fe8e-90ae-43b2-80b7-3701ee63e99f)
![obraz](https://github.com/community/community/assets/107455395/1e24908a-13ea-45b7-b82a-e312d2fe6790)

**Reply by ryukaizen** [reply]
Thank you!

**Reply by SebixDrela2** [reply]
You're welcome!

**Reply by rybak** [reply]
This is no longer available: https://github.com/orgs/community/discussions/58295#discussioncomment-7296655

**Comment by Undead34** [comment]
I did like the new interface, the colors are more modern and everything is cleaner and I think the space is much better distributed. <3

**Comment by matthew-e-brown** [comment]
I've already shared my feedback in the form linked in the sidebar, but I'll share again here. I really like the new navigation in the top right, my issue is with the per-page navigation. For things like profile pages and repositories, the tabs are ***way*** off to the top-left, so now the navigation for the main content is super disconnected. I think this is because the tabs for per-page navigation have been moved into the same navigation bar as the site-wide stuff, which is unintuitive.

I just got the update this morning and actually couldn't find the _Repositories_ tab for a second. Because the navigation is so disconnected, it looks like it would be navigation for the _entire site_, instead of this one page. So when I went to scan the page and find the button, my brain skimmed past it since, well, I don't want to navigate to elsewhere on the site, I want to navigate to elsewhere on this same page.

**Reply by matthew-e-brown** [reply]
Huh, I hadn't opened this menu till now:

![](https://github.com/community/community/assets/43387454/eff8f03c-f987-4a12-b7a1-ce70c5050386)

I don't see why this needed to change from a dropdown menu into a flyout that takes over the entire screen. Weird.

I **do** really like the new icons, though! 😄

**Reply by Undead34** [reply]
Haha LOL 😂🤣

**Comment by j-pedro-faria-alb** [comment]
why! oh why did you remove where i can jumpt to issues related to me? do I now have to navigate the sea of open issues? 
ive managed to use the links:
https://github.com/pulls/assigned
https://github.com/issues/assigned
Why? please put it back...

**Comment by aquanow-mario** [comment]
![image](https://github.com/community/community/assets/75330780/5e28beb7-a475-429f-85f6-afbbc6fc7ce4)

Removing the lines between each branch name list is giving me a hard time to accurately move my mouse cursor to where I wanted to. Moreover, it was not as fast as the previous UI was.

Here's the before UI screenshot:
![image](https://github.com/community/community/assets/75330780/04e689eb-f246-49a9-917f-bc0fd3080cb0)

**Comment by Gabe-Ambrosio** [comment]
Did they just force this UI change to be permanent? I had disabled the feature preview previously because the new UI is borderline unusable for me. But it looks like it's no longer listed in the feature previews. Am I part of a bad A/B test or is this permanent now?

**Reply by BitesizedLion** [reply]
Seems permanent yes, I despise whoever made this change

**Comment by reeveng** [comment]
I hate it, absolutely terrible

**Reply by reeveng** [reply]
First the homepage, now this? Who is behind all this?

**Reply by reeveng** [reply]
The previous UI I would use as an example of incredible UI, something every platform should strive for.
With this UI I am completely lost for words, truly missed the mark.

**Comment by sg-qwt** [comment]
Why am I being forced to this new UI again? No option to disable?

**Comment by Lunderberg** [comment]
For anybody else searching, the option mentioned in [this previous post](https://github.com/orgs/community/discussions/58295#discussioncomment-6243913) to disable the UI changes is no longer available.  The menu no longer includes the "Global Navigation Update", which could previously be used to re-enable the old interface.

![image](https://github.com/community/community/assets/3888575/0a349c00-2d43-48a2-8604-f1fc7038d51c)

I do hope that the option is brought back, as I prefer the old interface.  The words "Pull Requests" and "Issues" are much easier to find and identify than the low-contrast grey-on-grey icons in the top right of the new interface.

**Comment by AnthonyGrondin** [comment]
Agreed.

I find the new UI to be unintuitive and it forces me to relearn muscle memory for an interface I did not want, nor do I appreciate. I don't know how Github could mess this up, as the only thing they had to do, **was to do nothing**, but they still managed to fail miserably at it.

Removing features is not improvement.

**Comment by PickleNik** [comment]
I've provided tons of feedback during Feature Preview, all of which was ignored.
From jarring lack of drawer closing animations, to close buttons being on opposite sides for left and right drawers, dim theme overlay being too bright, having to manually scroll up every time...

Time to write an alt client for the Hub. 🤦‍♂️

**Reply by MidnightTinge** [reply]
And once alt clients get popular we'll get a prohibitive reddit-style API pricing structure to keep us in line 😄 I can't wait.

**Comment by v1993** [comment]
I've opted out back when this was provided for testing and left feedback about problematic parts. Now this change is permanent with literally nothing I've written about addressed (which is a lot, but search bar's behavior might be the worst offender for me personally). Much of interface feels like mobile UI patterns applied to desktop without much thought put behind them.

I'm not a big fan of GitLab's UI, but it's miles better than this "improvement". Unless there's an option to revert I'll be moving personal projects over there and limit GitHub usage to strict necessity.

**Comment by TheRealPear** [comment]
Not a fan of the new UI as it doesn't seem to be very colorful compared to the previous design (dark navigation bar in light mode), whereas it's all white in light mode. The menu bar feels like it was built for a large touchscreen (flyout menus, hamburger menu icons, etc.. instead of a traditional menu bar where you can see the links all the time). I really hope GitHub can consider reverting this, or at least redoing some aspects.

**Comment by aradalvand** [comment]
It's truly unbelievable just how irresponsible and dismissive of user feedback the GitHub UI design team have demonstrated themselves to be.

WE ABSOLUTELY HATE THE NEW DESIGN! And now you've forced it upon everyone! Fantastic! You have failed to address A SINGLE piece of feedback you were given.

**Comment by xamir82** [comment]
It's the worst thing I have ever seen... Whoever designed this shit (and removed the option to switch back to the old UI) should be jailed ASAP.
Pure incompetence, astonishing.

**Comment by eshack94** [comment]
I don't mean to be rude, but this UI update makes me wish I didn't have to use GitHub.

At least give devs back the _**option**_ to switch back and forth, or take user feedback into account prior to forcing this on everyone.

Edit: It looks like a lot of people gave great, detailed, pointed feedback about the new UI, which was all completely ignored.

**Comment by GeoffTearle** [comment]
Can we please reintroduce the ability to disable the new UI?

**Comment by tilda** [comment]
the post itself is not very constructive but there's definitely a reason why i personally disabled the changes when it was in preview

kind of a shame that github actively solicits feedback from users when disabling preview features yet *completely ignores what people say* and just pushes it out anyway...

**Comment by snowman** [comment]
don't take decision on behalf of users, you steal it, now return the option to turn it back

**Comment by zezhishao** [comment]
I think GitHub should give us a button to switch between the new UI and the old UI... instead of arrogantly making decisions for users.

**Reply by lolski** [reply]
I propose the button to be a one way switch from new to old UI

**Comment by 0xGhada** [comment]
# I hate any update that ```GitHub``` Makes it ... they should know we need a simple website to control and manage our projects, these updates are a shit

**Comment by 0xGhada** [comment]
# Why am I being forced to this new UI again?

**Comment by OscarTHZhang** [comment]
Why am I being forced to this new UI???? This is terrible and it's hard to find anything

**Comment by paddor** [comment]
The UI has also been feeling extremely slugglish. Terrible experience trying to take a quick look at a file.

**Comment by PickleNik** [comment]
why are they making the feed looking worse every other week 😹

**Comment by noroutine** [comment]
search ui is almost useless garbage and slow af, navigation is slow af

u need to find a line in a file, and you get one clusterfuck of jumping all around

I used to love gihub ui, now i hate it everytime i use it, it's confusing, overloaded, slow when it does help sometimes and stays in the way most of the time

**Comment by lolski** [comment]
As of today, the page will get slower and slower to a grinding halt when making a PR with lots of file changed, I had to wait around 15 seconds for the create draft PR button to respond (and I'm on a maxed out M1 MBP). The old pre-Microsoft UI would have had absolutely no problem displaying and responding to my request :)

**Comment by aradalvand** [comment]
It's been more than a year and a half and this update is still UTTER GARBAGE! Whoever designed it, and whoever let it go through won't see heaven.

**Comment by PickleNik** [comment]
@t3dotgg can you make a t3hub?

**Comment by Bewilderedornot** [comment]
Man, what a day. I got hit with the big PUK on my Samsung Galaxy S22 Ultra . So according to guidelines you know I'm just going to tell you how I really feel about it. Man I've and just supremely happy that people care so much because God knows I did not need any sleep at all and you know I didn't have to go do any school work of course because it just started right. So there would have been too much free time and I wouldn't have known what to do, luckily I'll just say from this predicament. I mean Class just started, and suddenly my phone's dead in the water. Apparently, my phone thought it was in New York, while my computer was chilling at home, and that flagged it as stolen. Poof! Locked out of everything – my textbooks, my GitHub key, everything on my phone. But I was in New York and my computer was at the house and I really do not understand how people do not understand that there are people that have to travel sometimes. But do you know if it wasn't for just almost losing everything on my phone including my new business and everything else and you know I would've had a really really relaxing day. I wouldn't know what to do you with those evil idle hands, everyone knows we can't have that in our society. But the culprit in this matter is not to be named although its not rocket science or computer science for that matter to figure out who they are.  I'm willing to help if you want me to help in any way to show you know what the rest of the world is you know feeling or doing or whatever you know, just let me know because a nobody from Arkansas. I don't know but I think my theory on this matter is correct...  just saying, if someone a lot more important gets hit with the P U K  on their lock screen, I'm pretty sure they would have taken it differently as I know you all would have also. I'm a fan of yours BTW, and I'm from Arkansas born and raised. But I thought I wouldn't be formal or anything, so I'm sure coming from a guy that English is his third language we should not be proper in this matter,  

From the beginning, I really looked up to you all and still do. It's amazing what you guys are doing. I just wanted to bring this to your attention. Also, for some reason a very popular map app decided that my address is no longer the same. This was a surprise to the postmaster and myself and all my financial means where conveniently store in your wallet. I have to once again remind you that the town I live in has approx. 3500 people. I was really late with tuition and bills in general, but that was last year...  I know its a lot of things you guys do, but without us to by and use your product...   I am not the sharpest tool, but that business model has just a few flaws....   So I got learned how to use Android Studio today and how to build a phone from scratch, so that was cool. This is the 2 time this happened this year...  and my card is jammed in my phone,  so taking it out wasn't going to work. Also the start up screen didn't connect to my computer after pushing a combination of the same buttons, because of course it didn't. But it is over for today so I can now worry about tomorrow. Loyalty goes a long way and I know its hard stopping all this malware and junk, but come on guy lay of the gas, you're hurting your own supporters.

Honestly, I would've been lost. I mean, who needs all that free time, right? Thankfully, I got to spend my entire day trying to fix it. So yeah, super lucky and just thrilled that people care so much. [](url)
![Gemini_Generated_Image_nid4wsnid4wsnid4](https://github.com/user-attachments/assets/eecf1784-f40b-4d4f-b689-051678eb79a1)

**Reply by PickleNik** [reply]
@grok please summarize this

**Reply by PickleNik** [reply]
oh wait wrong app my b
